### PR TITLE
JENKINS: Fix cuda module names

### PR DESCRIPTION
**Pull request by bureddy**

### CODE REVIEW

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 5 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 29-Nov-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/c5b0ca091b049c4fab0a9ecdf52fadb8c97e5e44/_main) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/761b965384c9ef35b9ced0edc6395beb43565bc4/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/32c47046ceeaa19103872136bb488d275f122b86/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/3eda760f58097c0e07bb328b257a9ca8b2978588/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/b5b9f1c924c853af6add0cd0baa06ad28b434494/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/5742d684bf3c23e3b3018f18440677a64a5e5b8d/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/8beeb29f7205402a55dd14f938add551e58cd5d5/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/d61d3ce8ad77e97ea0b03c6bb763f1b6126b36f9/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/e79b58536663c8c7c6c87b2a6db2d8bdb801150d/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/d2ec975a703a202e083b70b9264e9a944f439e1e/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/7125724a1cb167a3065a3925e36f1551f1dddaa1/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/57a3f2a5abefecc762834f6772a6cd00a7791283/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/00d6242242e2505ca0ccf89a79efde8511d3cabe/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/6a79065b421b1a353b63006c0b3acbcedf0f0c29/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/3202707c2055fd9ff3de2ec6fbbb069d20f5ddbf/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/5549380c1d27d62f22aad31874d1c7d1690ccb87/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/112e7af4cc36cea2cfdfa7e726c16d7a5b8b4a94/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/6541dcc69fcc92d193fdfb2d02589d8ccd5ec43b/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/027c08b4d66d1082788ecb8631d0c8cb0f96ddbf/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/6498ad9968a036db47809dd91abd4652721cae9e/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/1a5b1053b1c5b19d683aad60e605394648b65ad7/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/1fa40642c94f6565c9f7bbaa3ed4d39e8da8443e/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/5ec3e876581495379edb64f5bb728d843705b233/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/880779463bccd129f96c891b863e2e5977bcf290/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1ba9ea59014d9f2edc112200cfffdf1c/raw/87ac36b582f659075ee3c31dfd318f19515f1958/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by bureddy** [comment]
```
01:28:44 /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-test-node-gpu/worker/0/contrib/../src/uct/rocm/gdr/rocm_gdr_ep.c: In function ‘uct_rocm_gdr_ep_put_short’:
01:28:44 /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-test-node-gpu/worker/0/contrib/../src/uct/rocm/gdr/rocm_gdr_ep.c:43:9: error: implicit declaration of function ‘gdr_copy_to_bar’ [-Werror=implicit-function-declaration]
01:28:44          ret = gdr_copy_to_bar((void *)remote_addr, buffer, length);
```

 @paklui @yuq  rocm gdr_copy build is failing after upgrading nvidia gdr copy userspace to 2.0.  is RoCM expected to work with new NVIDIA gdrcopy userspace library?  Can you please look into it?

@yosefe will stick to gdrcopy-1.3 in Jenkins until this is resolved.

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 29-Nov-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/8004544d4971e6c075981342f7e3bfd0a181a55c/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/76f272e2aafd6b19be4c717c4d4043072299607e/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/5a6b3af0052fddb45a5237aea4d68dee592fd3b9/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/4a2f4e2ab2df8b7bc6756f3d2d3febd5235573e4/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/29b6481f155ab66b73500444886db32c88af1328/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/f84e423afb80da031ae98fd1c0e2033b4b6795f9/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/3149b7f49689c8b4790e356114621b7c597487b3/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/29d1b960ccdbed4ca4c726fc8bdcdda10cb32e89/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/085bf6d600a1377ab1e51c73ac5a4d0d19be3377/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/3f9eda07e77e5a524ce57ec8ddbdbfef82e4f2c0/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/687bccad00ad53bb77de88c23acc967909304fa2/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/ab54f367b5a9a34b6472cc7503f7d86fd3409cc9/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/d08db3e657fb6e77223171949e769d13456af775/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/8acffde8cafdc8db339f0f5286dd02560112b7bc/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/bcf9a737522800f3f4cc0a7d9792f6ab77f69f7c/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/6813c7caf20a6ee2118c11a658c208156b6e0e16/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/3169ea961ed7b7a047c62eab64818593a1cce413/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/a73d7fea62173c91ed028c71dee7d89a313eba1a/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/311a95f2692d0be2c89ce0f760b63e6584116b31/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/d5a7442f0bee5b1cea5cbcb90091fcda73e8e808/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/31bb2ea8a6a98ba4ac24db703ac6f361899a47a1/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/1794147424e7db2e3c96d6c03a9df7923a610be7/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/a111223a78346aaab9393fea24ced6c20f88abe3/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/d73e780980187e59fac20f15de4b37ad51f44e3d/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/076eae05792926e7b0692c2787572b5b/raw/39711dde7b8dc9f275d68a5bd29dddcd5342d61f/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -24,8 +24,8 @@
 
 WORKSPACE=${WORKSPACE:=$PWD}
 ucx_inst=${WORKSPACE}/install
-CUDA_MODULE="dev/cuda10.1"
-GDRCOPY_MODULE="dev/gdrcopy"
+CUDA_MODULE="dev/cuda-latest"
+GDRCOPY_MODULE="dev/gdrcopy1.3_cuda10.1"
```
Comment (yosefe) [inline contrib/test_jenkins.sh]: cuda is latest but gdr is specific version.. should have gdr-latest as well..

Comment (bureddy) [reply contrib/test_jenkins.sh]: RoCM gdr copy build is failing with the latest gdr copy. (https://github.com/openucx/ucx/pull/4488#issuecomment-557395719).  I opened an issue for it (https://github.com/openucx/ucx/issues/4489).  we need some clarification from AMD. 
I would like go with this specific version to enable the cuda testing. cuda tests are disabled after cuda module names changed in the environment. Would like to merge this asap  enable the cuda testing again.

Comment (yosefe) [reply contrib/test_jenkins.sh]: @bureddy i've pushed commit to this PR to load Cuda by specific version as well, for consistency
we can change both to -latest when RoCM issue is fixed

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 5 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 01-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/d20562f337bb867ce724e8cd209d8ce3168e566e/_main) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/dfd681c8f22f0596d45d731ffbc5bdc56891c09b/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/f5fe09280ec13edc86b32ae6e6b98117121da263/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/9dff6d88dd1d792c8768bcc8ac74f47997a78d59/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/c7fff26cd386adec64156f52bbf0fb284f07b89e/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/059476043c7a3972e29a53a3726badb41350d16d/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/6af50b0d00bdd63a9136b96b0717105745199f3c/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/8045d761fc924fafb02e634d47cd0dc262acc10a/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/333527f345bdcd9389bb9e37b38f124756bef323/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/eeec0374c24607a55852a94e02dd3246af6cb541/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/fab4fef9d01ea92025421056128184410ea77bcc/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/c4840facb9b24fed82f8113a69b99a83378ad076/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/e4063b1d355a4eb96eace6757ed85abf5b6df810/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/1e187e9f17a4798fb49247b2592535a50f51131e/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/743b6a541ea2f53d0bb61468d646689ffdb88534/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/0d10113c8dd127cfef69a6a215bc0059737248ef/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/98c8e46e0ef6d4153a7a1c0b43d73e4322276763/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/eac9caa1c2541d58cd6de9bd63625301dfafb34e/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/f51181786a794a522cd2286d44c419b2bce28d66/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/3076d11ad82d460ac4a203cde8e3f00ac840870a/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/27a5b7b338d1f25fca56ec333eaeb16230d77664/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/696c97a281fa0edb615dc12a1826d595c53c5300/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/610c94dd88efd1da2ce743546f444896a0fe84a7/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/76ebc98392f1f28b805410e5d2b2d18f2e19a3bc/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/927e8c42df5d7443e3c62221feb17614/raw/b069dd8fd718b9941e6a19fe55fecbf7b915412e/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
depends on #4501

**Comment by yosefe** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 02-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/6474f98db96eed4576794d057a6c67f31fecfe24/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/74c05f8aa7cef4558e86340aaf8b8da674b9ccc4/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/99ce91467ab98c63cd5bd04356cfe549d6043c15/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/41823e1822fd5b0fdf421f384fa3966de5436042/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/53d28f1104c3c1c3627ffe67b27cb247f93c568e/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/7dd7d50b07111992dfe3a855c10c84d56a31b636/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/734d339cd802b41fd001493b6fd486b0bb186667/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/14e1d54b933844c5e73f088899ffd07010c26387/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/33565ec58c898344316dfd01733ecc9ba1354d50/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/77fe91a15bc18afe966114312cca6bc7713b088f/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/f05a79849d5577d9e1d38ceef055aa6fd72d6e57/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/170f2790c7d5f74c782c5de1acce208febf4cf36/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/caf63d1ffa15adf8da7ee30dcbea25552e9482b6/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/ed6983d1247466eb4cf0004081174f6a67c08e62/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/c04d52822347db375ffbf9119df6aad3afd2ffd2/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/abdc4e10a70e215e15c872fd57eb3c76c8be75d8/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/ca47a6b4bad5a4d38ac057e6a31b372b500580e8/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/986f07009e00c416c525bcbdfe71985adeb7b104/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/81a736feaaa2c814c5137b93fba193a7cef4af43/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/4482b90dfa422c80dc720aeb352fb3305b4d9153/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/fd7bea1f9ec123d71815e13a5981a214a67c2508/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/f4804c76839444787244cd522d6b50d5c1c4203d/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/7dbdedfe3f5bc96702ea724fc84734915f56800a/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/3384cde01d8608c46d496c771ea18122f0783a1b/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/64ebf696c17760054779ad1e1dbcb1bd/raw/246147599d553028c9324e4cbc651dc6eb984dd3/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**